### DISCUSSION TITLE
save all accounts in tdata to session

### DISCUSSION THREAD

**Discussion by quessy-dev** [thedemons/opentele#76] [General]
Hi! I learning your library and i have 1 problem:
How to save sessions if there is more than 1 account in tdata?
Thanks in advance!

**Comment by thedemons** [comment]
By default, `TDesktop.ToTelethon` will use the main account - the account at index 0. You can use other accounts by using `TDesktop.accounts[]` with the index from 0 to 2:
```python
tdesk = TDesktop(tdataFolder)
tdesk.accounts[0].ToTelethon(...)
tdesk.accounts[1].ToTelethon(...)
tdesk.accounts[2].ToTelethon(...)
```

**Reply by exaho** [reply]
> By default, `TDesktop.ToTelethon` will use the main account - the account at index 0. You can use other accounts by using `TDesktop.accounts[]` with the index from 0 to 2:
> 
> ```python
> tdesk = TDesktop(tdataFolder)
> tdesk.accounts[0].ToTelethon(...)
> tdesk.accounts[1].ToTelethon(...)
> tdesk.accounts[2].ToTelethon(...)
> ```

How abou 3 session to 1 tdata? Is that possible sir?
